AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -247,7 +248,7 @@ - (NSString *)convertTokenToString:(NSData *)token {
 - (void)sendPushToken:(NSString *)token {
   MSPushLog *log = [MSPushLog new];
   log.pushToken = token;
-  [self.channelUnit enqueueItem:log];
+  [self.channelUnit enqueueItem:log flags:MSFlagsNone];
```

Comment:
Should it be "Normal"?

Reply:
None needs to map to default values for each flags as discussed.